### ISSUE TITLE
Title: Reproducibility issue — low performance on Market1501 (mAP 36.1%, Rank-1 61.4%)

### ISSUE THREAD

**Issue by joohyung0809** [songchunli1999/MEDA#1]
---

**Description**

I tried to reproduce the results but obtained much lower performance on Market1501. I set up a federated-by-camera experiment (each camera = client) and ran the code with the following details; hoping someone can point out what I might be doing wrong.

**What I ran / settings**

* Dataset: Market1501
* Federated setting: each camera is a client (federated-by-camera)
* Test filtering: excluded gallery entries with label `-1`
* Rounds: set to 300 (same as paper)
* Everything else left at default from the repo

**Code change I made**
In `client.py` there was a line:

```py
out_feature1 = self.model(inputs)
```

But `model.forward` actually returns `(x, out_feature1)`, so I changed it to:

```py
_, out_feature1 = self.model(inputs)
```

(That is the only code change I made.)

**Observed results**

* mAP: **36.1%**
* Rank-1: **61.4%**

These are far from the numbers reported in the paper.

**What I checked / notes**

* I removed gallery `-1` entries before testing (as noted above).
* I used 300 rounds per the paper.
* No other changes to training/evaluation code or hyperparameters.

**Request / questions**

* Could this mismatch be caused by my single-line change to unpack the model output, or is there another common pitfall I should check (data preprocessing, model.eval()/train mode, feature normalization, multi-scale test flag, BN behavior across clients, checkpoint loading, etc.)?
* Are there any repo-specific evaluation details or preprocessing steps (e.g., resize/crop, normalization, use of `extract_feature` multi-scale or normalization options) I might have missed?

Thanks in advance for any pointers — I can attach logs and code snippets if useful.
